subsys/net/ip/rpl.c

```diff
@@ -2044,7 +2044,7 @@ static bool net_rpl_process_parent_event(struct net_if *iface,
 	return ret;
 }
 
-static bool net_rpl_repair_root(u8_t instance_id)
+bool net_rpl_repair_root(u8_t instance_id)
```

Comment:
This is related to #980 RPL border router sample application where user can manually trigger repair.